### PR TITLE
Add outbound tracking on links

### PR DESCRIPTION
**Pull request by smythp**
## Description

Adding outbound tracking on links to our main CTAs, such as links to the images directory.

## Reviewing

I ran a script that uses regex, it would be good to check for edge cases. I'll also do a few manual edits, I'll keep those in separate commits. The commit name will show if it's manual or automated.

## Notes

I added a content field so we know what page is generating the outbound link in analytics. I've done some stuff so that it's not a useless path like "index".

Resolves  https://github.com/chainguard-dev/internal/issues/4233

cc @sheesh @ltagliaferri  @SharpRake

### CODE REVIEW

**Comment by smythp** [comment]
Added these links to our rumble comparison pages:

```
https://images.chainguard.dev/directory/image/{{ image_name }}/overview?utm_source=cg-academy&utm_medium=website&utm_campaign=dev-enablement&utm_content=cg-academy-rumble-comparison-{{ image_name }}
```

**Review by amouat — APPROVED** [review]
looks good, just double check /s

File: content/chainguard/administration/iam-organizations/identity-examples/gitlab-identity.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ assume-and-explore:
 
 Let's go over what this configuration does.
 
-First, GitLab requires that pipelines have a shell. To this end, this configuration uses the [`cgr.dev/chainguard/wolfi-base` image](https://images.chainguard.dev/directory/image/wolfi-base/overview) since it includes the `sh` shell.
+First, GitLab requires that pipelines have a shell. To this end, this configuration uses the [`cgr.dev/chainguard/wolfi-base` image](https://images.chainguard.dev/directory/image/wolfi-base/overview?utm_source=cg-academy&utm_medium=website&utm_campaign=dev-enablement&utm_content=chainguard/administration/iam-organizations/identity-examples/gitlab-identity) since it includes the `sh` shell.
```
[the remark this answers is not in the document]
Comment (smythp) [reply content/chainguard/administration/iam-organizations/identity-examples/gitlab-identity.md:241]: Yeah that checks out. I'll replace them with hyphens. They don't need to be machine readable anyway.

**Comment by smythp** [comment]
On pause while I get a little review on the utm fields

File: content/chainguard/administration/iam-organizations/identity-examples/gitlab-identity.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amouat) [reply content/chainguard/administration/iam-organizations/identity-examples/gitlab-identity.md:241]: I looked it up out of curiosity. You shouldn't use `:/?#[]@!$&'()*+,;=`. Hyphen is fine.

**Pull request merged**